src/packages/PkgCompat.jl

```diff
@@ -194,6 +194,30 @@ function update_registries(; force::Bool=false)
 	end
 end
 
+# ✅ Public API
+"""
+Check when the registries were last updated. If it is recent (max 7 days), `Pkg.UPDATED_REGISTRY_THIS_SESSION` is set to `true`, which will prevent Pkg from doing an automatic registry update.
+
+Returns the new value of `Pkg.UPDATED_REGISTRY_THIS_SESSION`.
+"""
+function check_registry_age(max_age_ms = 1000.0 * 60 * 60 * 24 * 7)::Bool
```

Comment:
```suggestion
function check_registry_age(max_age_ms = 1000.0 * 60 * 60 * 24 * 2)::Bool
```
Maybe 2 days? 7 is quite long maybe

Reply:
Yeah, I wonder what this is like for people who do not work on Julia packages...

Reply:
Good question. I was also thinking about people who get confused about the fact that some package has updated but doesn't want to update inside Pluto.jl because the registry is outdated

Reply:
clicking the update button (Pkg.update) will always update the registry i think... should check that

Reply:
Yep!

```
(@v1.7) pkg> activate --temp
  Activating new project at `/var/folders/v_/fhpj9jn151d4p9c2fdw2gv780000gn/T/jl_i0FO8a`

(jl_i0FO8a) pkg> update
    Updating registry at `~/.julia/registries/PlutoPkgTestRegistry`
    Updating git-repo `/Users/fons/Documents/testregistry/PlutoPkgTestRegistry`
    Updating registry at `~/.julia/registries/General.toml`
  No Changes to `/private/var/folders/v_/fhpj9jn151d4p9c2fdw2gv780000gn/T/jl_i0FO8a/Project.toml`
  No Changes to `/private/var/folders/v_/fhpj9jn151d4p9c2fdw2gv780000gn/T/jl_i0FO8a/Manifest.toml`
[ Info: We haven't cleaned this depot up for a bit, running Pkg.gc()...
      Active manifest files: 84 found
      Active artifact files: 258 found
      Active scratchspaces: 33 found
     Deleted 59 package installations (83.974 MiB)

(jl_i0FO8a) pkg> update
    Updating registry at `~/.julia/registries/PlutoPkgTestRegistry`
    Updating git-repo `/Users/fons/Documents/testregistry/PlutoPkgTestRegistry`
    Updating registry at `~/.julia/registries/General.toml`
  No Changes to `/private/var/folders/v_/fhpj9jn151d4p9c2fdw2gv780000gn/T/jl_i0FO8a/Project.toml`
  No Changes to `/private/var/folders/v_/fhpj9jn151d4p9c2fdw2gv780000gn/T/jl_i0FO8a/Manifest.toml`

```